bin/lint-markdown-api-history.ts

```diff
@@ -237,6 +247,34 @@ async function main(
           }
         }
 
+        // Throw an error if a description isn't surrounded by double quotation marks
+        if (checkDescriptions) {
+          const possibleDescription = codeBlock.value.matchAll(possibleDescriptionRegex);
+
+          for (const [matchedLine, matchedGroup] of possibleDescription) {
+            const trimmedMatchedGroup = matchedGroup.trim();
+            const isMatchedGroupInsideQuotes =
+              trimmedMatchedGroup.startsWith('"') && trimmedMatchedGroup.endsWith('"');
+
+            if (isMatchedGroupInsideQuotes) continue;
+
+            console.error(
+              'Error occurred while parsing Markdown document:\n\n' +
+                `'${filepath}'\n\n` +
+                'Possible description field is not surrounded by double quotes.\n\n' +
+                'This might cause issues when parsing the YAML (might not throw an error)\n\n' +
+                'Matched group:\n\n' +
+                `${matchedGroup}\n\n` +
+                'Matched line:\n\n' +
+                `${matchedLine}\n\n` +
+                'API history block:\n\n' +
+                `${possibleHistoryBlock.value}\n`,
+            );
+            errorCounter++;
+            continue;
```

Comment:
I assume this `continue` is meant to continue to the next API history block like the others, but in this case it's the last line inside a for-loop so it will have no effect. A slight refactor will be needed to skip to the next API history block.

Reply:
Good catch! Would this be ok?

```patch
diff --git a/bin/lint-markdown-api-history.ts b/bin/lint-markdown-api-history.ts
index dc1f35b..f24b31f 100644
--- a/bin/lint-markdown-api-history.ts
+++ b/bin/lint-markdown-api-history.ts
@@ -168,7 +168,7 @@ async function main(
       }
     }
 
-    for (const document of await workspace.getAllMarkdownDocuments()) {
+    documentForLoop: for (const document of await workspace.getAllMarkdownDocuments()) {
       const uri = URI.parse(document.uri);
       const filepath = workspace.getWorkspaceRelativePath(uri);
 
@@ -271,7 +271,7 @@ async function main(
                 `${possibleHistoryBlock.value}\n`,
             );
             errorCounter++;
-            continue;
+            continue documentForLoop;
           }
         }
 
```